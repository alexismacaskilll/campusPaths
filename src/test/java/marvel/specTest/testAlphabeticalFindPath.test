# Test the BFS picks alphabetically least path on a file that lists the heros and books
# in reverse alphabetical order

# Load a graph from tsv file that contains heros and comics in reverse alphabetical order.
LoadGraph graph1 alphabetical.tsv

# Find shortest path from hero1 to hero4
FindPath graph1 hero1 hero4