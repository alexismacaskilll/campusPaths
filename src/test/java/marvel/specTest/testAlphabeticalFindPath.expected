# Test the BFS picks alphabetically least path on a file that lists the heros and books
# in reverse alphabetical order

# Load a graph from tsv file that contains heros and comics in reverse alphabetical order.
loaded graph graph1

# Find shortest path from hero1 to hero4
path from hero1 to hero4:
hero1 to hero2 via book1
hero2 to hero4 via book3